storage/local/storage.go

```diff
@@ -212,10 +212,18 @@ func (s *memorySeriesStorage) Start() error {
 		return err
 	}
 	s.persistence = p
+	go s.persistence.run()
```

Comment:
Perhaps add a comment that persistence needs to run before p.loadSeriesMapAndHeads() is called.